granian/cli.py

```diff
@@ -194,6 +194,38 @@ def option(*param_decls: str, cls: Optional[Type[click.Option]] = None, **attrs:
     default=False,
     help="Enable auto reload on application's files changes (requires granian[reload] extra)",
 )
+@option(
+    '--reload-paths',
+    type=click.Path(exists=True, file_okay=True, dir_okay=True, readable=True, path_type=pathlib.Path),
+    default=(pathlib.Path.cwd(),),
+    help='Paths to watch for changes',
+    multiple=True,
+)
+@option(
+    '--reload-ignore-dirs',
+    default=(),
```

Comment:
If the option has no default value, I would just skip the whole `default=` thing, just to be consistent with the rest of the code.

Reply:
Done!